.travis.yml

```diff
@@ -1,71 +1,47 @@
 language: python
+cache: pip
+
 python:
-  - "2.6"
   - "2.7"
-  - "3.4"
-  - "3.5"
   - "3.6"
-install:
-  - pip install coveralls coverage
-  - pip install $DJANGO
-script:
-  - coverage run runtests.py
-  - coverage report -m
-after_script:
-  - coverage combine
-  - coveralls
+  - "3.7"
+
+sudo: false
+
 env:
-  - DJANGO="Django>=1.4,<1.5"
-  - DJANGO="Django>=1.5,<1.6"
-  - DJANGO="Django>=1.6,<1.7"
-  - DJANGO="Django>=1.7,<1.8"
   - DJANGO="Django>=1.8,<1.9"
   - DJANGO="Django>=1.9,<1.10"
   - DJANGO="Django>=1.10,<1.11"
   - DJANGO="Django>=1.11,<2.0"
+  - DJANGO="Django>=2.0,<2.1"
+  - DJANGO="Django==2.2.12"
   - DJANGO="https://github.com/django/django/archive/master.tar.gz"
+
 matrix:
+  fast_finish: true
   exclude:
-    # Python 2.6 support has been dropped in Django 1.7
-    - python: "2.6"
-      env: DJANGO="Django>=1.7,<1.8"
-    - python: "2.6"
-      env: DJANGO="Django>=1.8,<1.9"
-    - python: "2.6"
-      env: DJANGO="Django>=1.9,<1.10"
-    - python: "2.6"
-      env: DJANGO="Django>=1.10,<1.11"
-    - python: "2.6"
-      env: DJANGO="Django>=1.11,<2.0"
-    - python: "2.6"
-      env: DJANGO="https://github.com/django/django/archive/master.tar.gz"
+    - python: "2.7"
+      env: DJANGO="Django>=2.0,<2.1"
+    - python: "2.7"
+      env: DJANGO="Django==2.2.12"
```

Comment:
Python 2.x is no longer supported. 

[Source](https://www.python.org/dev/peps/pep-0373/)